packages/fether-electron/src/main/app/options/config/index.js

```diff
@@ -79,4 +126,81 @@ const TASKBAR_OPTIONS = {
   withTaskbar: true
 };
 
-export { DEFAULT_OPTIONS, TASKBAR_OPTIONS };
+const SECURITY_OPTIONS = {
+  // Custom network settings
+  network: {
```

Comment:
sorry i'm not sure what your concern is? i was trying to keep the config together in the same place.  i'll look into it again and see if i can find any issues or alternatives

Reply:
My concern is this line:

https://github.com/paritytech/fether/blob/1a0a15ebd831a0147a2055e3dd150c4eb26abfad/packages/fether-electron/src/main/app/methods/createWindow.js#L25

Inside `options`, there's `SECURITY_OPTIONS`, to which you just added a `network` field which is not part of what `BrowserWindow` is expecting.

As I said, there's 99.9% chance `BrowserWindow` will just ignore this `network` field, so we can leave it like this. Maybe we can add a comment above here?

Reply:
Thanks. I didn't think it'd cause any harm but there's still a possibility it may expose us to a name conflict if the Electron already has or adds that field name in future to `BrowserWindow`, so i'll add a comment and change it to a field name like `fetherNetwork` that's less prone to potential name conflicts from Electron updates